ledger/complete/ledger_test.go

```diff
@@ -355,6 +355,75 @@ func TestLedgerFunctionality(t *testing.T) {
 	}
 }
 
+func Test_ExportCheckpointAt(t *testing.T) {
```

Comment:
Maybe we can add a bit of documentation here, indicating that the state of the initial ledger, i.e. has 0->"A" and 1->"B"

Reply:
Added comments at c3cc915